java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/protocol/Server.java

```diff
@@ -195,6 +196,9 @@ synchronized void attachClient(NbCodeLanguageClient client) {
         public MessageConsumer attachLookup(MessageConsumer delegate) {
             // PENDING: allow for message consumer wrappers to be registered to add pre/post processing for
             // the request plus build the request's default Lookup contents.
+            if (!(delegate instanceof Endpoint)) {
```

Comment:
Thanks for catching this! I just didn't get right the idea that lsp4j sends BOTH the incoming and outgoing messages go throught the same wrapper through my head :(